diff -crB etc/apache2/ports.conf apache2.modified/ports.conf
*** etc/apache2/ports.conf	2024-10-26 07:51:14.076102563 -0400
--- apache2.modified/ports.conf	2024-10-26 07:51:25.599838378 -0400
***************
*** 2,8 ****
  # have to change the VirtualHost statement in
  # /etc/apache2/sites-enabled/000-default.conf
  
! Listen 80
  
  <IfModule ssl_module>
  	Listen 443
--- 2,8 ----
  # have to change the VirtualHost statement in
  # /etc/apache2/sites-enabled/000-default.conf
  
! Listen ${HTTP_PORT}
  
  <IfModule ssl_module>
  	Listen 443
diff -crB etc/apache2/sites-available/000-default.conf apache2.modified/sites-available/000-default.conf
*** etc/apache2/sites-available/000-default.conf	2024-10-26 07:51:14.077102540 -0400
--- apache2.modified/sites-available/000-default.conf	2024-10-26 07:51:48.936303888 -0400
***************
*** 1,4 ****
! <VirtualHost *:80>
  	# The ServerName directive sets the request scheme, hostname and port that
  	# the server uses to identify itself. This is used when creating
  	# redirection URLs. In the context of virtual hosts, the ServerName
--- 1,4 ----
! <VirtualHost *:${HTTP_PORT}>
  	# The ServerName directive sets the request scheme, hostname and port that
  	# the server uses to identify itself. This is used when creating
  	# redirection URLs. In the context of virtual hosts, the ServerName
Only in apache2.modified/sites-enabled: 000-default.conf
